tests/commands/test_common_command.py

```diff
@@ -0,0 +1,69 @@
+import sys
+
+import pytest
+from pytest_mock import MockFixture
+
+from commitizen import cli
+from commitizen.commands import Example, Info, ListCz, Schema
+
+
+@pytest.mark.skipif(
+    sys.version_info < (3, 13),
+    reason="The output message of argparse is different between Python 3.13 and lower than Python 3.13",
+)
+@pytest.mark.parametrize(
+    "command",
+    [
+        "bump",
+        "changelog",
+        "check",
+        "commit",
+        "example",
+        "info",
+        "init",
+        "ls",
+        "schema",
+        "version",
+    ],
+)
+def test_command_shows_description_when_use_help_option(
+    mocker: MockFixture,
+    capsys,
+    file_regression,
+    monkeypatch: pytest.MonkeyPatch,
+    command: str,
+):
+    """Test that the command shows the description when the help option is used.
+
+    Note: If the command description changes, please run `pytest tests/commands/test_common_command.py --regen-all` to regenerate the test files.
+    """
+    # Force consistent terminal output
+    monkeypatch.setenv("COLUMNS", "80")
+    monkeypatch.setenv("TERM", "dumb")
+    monkeypatch.setenv("LC_ALL", "C")
+    monkeypatch.setenv("LANG", "C")
+    monkeypatch.setenv("NO_COLOR", "1")
+    monkeypatch.setenv("PAGER", "cat")
```

Comment:
I think this could be a fixture like `fixed_term` for any test relying on output.
Basically, what it does:
- `COLUMNS` fix the width of the term
- `TERM=dumb` deactivate reactive output on libs supporting this (like spinners, progress bars...), better for static snapshots.
- `LC_ALL=C` and `LANG=C` disable localization and utf-8 chars in most CLI tools, it ensure test snapshot doesn't fail on non-english setups 
- `NO_COLOR` is the now "standard" way of disabling ANSI color support (cf. https://no-color.org/)
- `PAGER=cat` will avoid user defined pagers like `bat` to change the output.